src/metaswitch/crest/api/homestead/__init__.py

```diff
@@ -171,12 +130,8 @@
     (r'/public/'+ANY+r'/associated_private_ids/?', PublicIDPrivateIDHandler),
 ]
 
-ROUTES = CACHE_ROUTES
-if settings.LOCAL_PROVISIONING_ENABLED:
```

Comment:
This setting should probably be removed from everywhere in the project, since I don't think it's used any more, and I don't think it makes any sense any more.